intl/msg_hash_us.h

```diff
@@ -1443,11 +1443,11 @@ MSG_HASH(
    )
 MSG_HASH(
    MENU_ENUM_SUBLABEL_CRT_SWITCH_HIRES_MENU,
-   "Use high resolution menu"
+   "Use a high resolution menu betweeeon content. Allows use of other menu drivers"
```

Comment:
Apart from the obvious spelling mistake (spell checkers are a great help, btw), I still have questions. What do you mean by "Use a menu _between content_"? Or "Allows use of other menu drivers"? You have to keep in mind, that many people will need to understand these sentences, to be able to provide a proper translation. So it's better to be overly detailed rather than too condensed.